pip/cryptography.file

```diff
@@ -0,0 +1 @@
+Requires: py3-setuptools-rust py3-cffi
```

Comment:
please also add
```
%define PipPreBuild export CARGO_HOME=%{_tmppath}/cargo_home
```
this will allow cargo to use our buidl area instead of `~/.cargo` directory